pkg/sbom/spdx/marshal.go

```diff
@@ -336,6 +338,14 @@ func (m *Marshaler) pkgToSpdxPackage(t, pkgDownloadLocation string, class types.
 		return spdx.Package{}, xerrors.Errorf("package file error: %w", err)
 	}
 
+	var supplier *spdx.Supplier
+	if pkg.Maintainer != "" {
+		supplier = &spdx.Supplier{
+			SupplierType: PackageSupplierNoAssertion,
+			Supplier:     pkg.Maintainer,
+		}
+	}
+
```

Comment:
`SupplierType` should be empty when Supplier is `NOASSERTION`.
https://github.com/spdx/tools-golang/blob/ee0b98bce00f50878120ac3ba725ba0396d6cfb7/spdx/v2/common/package.go#L11-L16

And we may want to use "Organization" for maintainers.

```suggestion
	supplier := &spdx.Supplier{Supplier: PackageSupplierNoAssertion}
	if pkg.Maintainer != "" {
		supplier = &spdx.Supplier{
			SupplierType: "Organization", // Always use "Organisation" at the moment as it is difficult to distinguish between "Person" or "Organization".
			Supplier:     pkg.Maintainer,
		}
	}
```

Reply:
i tried to use empty `SupplierType`, but `tools-golang` returns error for this case. 
that is why i used `NOASSERTION`.

But okay, let's change that.

Reply:
Done